app/src/main/java/cm/aptoide/pt/view/swipe/LoaderLayoutHandler.java

```diff
@@ -30,10 +31,7 @@
   private final List<Integer> viewsToShowAfterLoadingId = new ArrayList<>();
   protected ProgressBar progressBar;
   private List<View> viewsToShowAfterLoading = new ArrayList<>();
-  private View genericErrorView;
-  private View noNetworkConnectionView;
-  private View retryErrorView;
-  private View retryNoNetworkView;
+  private ErrorView errorView;
```

Comment:
Not being unbinded. I know that the other views are not being unbinded as well and you are not the one responsible for this, but do you know why ?

Reply:
Only God knows 🙏 

But seriously, I don't, someone probably forgot.